src/style_builder.cr

```diff
@@ -192,17 +192,17 @@ module Mint
       selectors =
         ["." + style_pool.of(node, nil)]
 
-      process(node.body, nil, nil, selectors, [] of String, node)
+      process(node.body, nil, nil, selectors, %w[], node)
     end
 
     # Processes a Ast::CssSelector
     def process(node : Ast::CssSelector,
-                id : String | Nil,
-                at : String | Nil,
+                id : String?,
+                at : String?,
                 parents : Array(String),
                 conditions : Array(String),
                 style_node : Ast::Node)
-      selectors = [] of String
+      selectors = %w[]
```

Comment:
just an opinion here but `[] of String` is more readable to newcomers

Reply:
Perhaps, but that's what the language reference is for, I don't see a point in using only newcomers-friendly syntax, it ain't a language playground.